app/scripts/components/common/layout-root/index.tsx

```diff
@@ -98,7 +104,11 @@ function LayoutRoot(props: { children?: ReactNode }) {
           />
         )}
       </PageBody>
-      <PageFooter hideFooter />
+      {isUSWDSEnabled ? (
```

Comment:
This all looks good! I may have missed some context from the header nav work, but what is the reason we are hiding the new refactor work behind a **isUSWDSEnabled** flag?

Reply:
Is this related to the USWDS token work and requires additional implementation steps from the instances?

Reply:
We discussed this verbally in our devsync.